testdata/acquisition/covid_hosp/state_daily/metadata.csv

```diff
@@ -0,0 +1,9 @@
+Update Date,Days Since Update,User,Rows,Row Change,Columns,Column Change,Metadata Published,Metadata Updates,Column Level Metadata,Column Level Metadata Updates,Archive Link
```

Comment:
This appears to be the facility metadata, not the state metadata.

Reply:
replaced with state metadata